### DISCUSSION TITLE
Bauhaus Great Infurior (BAUHAUS) [R]

### DISCUSSION THREAD

**Discussion by 21-ric** [secretcowlevel/DoomtrooperPublic#11] [Card Balance]
I have 2 proposals for this card.

1) Update this card to be playable.

Bauhaus Great Infurior (BAUHAUS) [R]
INFLUENCE
 As 1 Action and 6 [DP] , you may destroy any SUPPORT. 

2) Remove the card from the collection. We'll bring back this card with fortification.

**Comment by Kite80** [comment]
Compared to similar cards like supply lines evered, a reasonable cost would be 2 actions + 5 DP; less than this would be too cheap since it virtually has unlimited use because is not discarded after use or his cost doesn't raise after each shot

**Reply by Sol-au-fein** [reply]
Compared to support destroying tools available to other factions, DP cost is unnecessary and overpriced. Still It is better to put cost on activation than on deploying.

**Comment by Sol-au-fein** [comment]
Restricted. Unique.
Deploy 1 action. Destroy target support 3 action. No DP cost. 
Compared to Capitol rocket launcher, which is only 1 tactical action and you can have 3 of them, It is still inferior.

**Comment by 21-ric** [comment]
"Leveled field" 3 actions : all cards are discarded except warriors
"supply lines severed" 2 actions : 1 INFLUENCE card is discarded 
"DPAT-9" 1 tactical action to destroy any SUPPORT
"Equilibrium" 1;2;3;4;5 actions and 1 BROTHERHOOD warrior to discard 1 INFLUENCE or 1 CONDITION
"Curse of Alegroth" 0;1;2;3 actions to discard 1 INFLUENCE or 1 CONDITION

All cards destroy "Bauhaus great infurior".
@Kite80 : you compare only with "supply line severed".
@Sol-au-fein : your proposal is similar to "special forces training" for attachment. 
Why "special forces training" is not played? Because it's expensive. :walking:

**Reply by Sol-au-fein** [reply]
5DP or 3 actions, both is expensive.

**Reply by 21-ric** [reply]
@Sol-au-fein 
Bauhaus Great Infurior (BAUHAUS) [R]
INFLUENCE
Play for 2 Actions.
As 2 Action, you may destroy any SUPPORT.

Is it better for you? 4 actions to destroy any Support (3 with building boom). After it's 2 actions.

**Reply by Sol-au-fein** [reply]
Playable

**Reply by Kite80** [reply]
what about deploy for 1 action and 5DP, fire for 2 actions? 1st turn it comes to play it can already be used and destroy 1 support; maybe it becomes a little OP if thre are also bauhaus relic and building boom...

**Reply by 21-ric** [reply]
If we add a DP cost for deploy this card, we'll kill "again" this card. 

1 action to attach +3 actions and 5DP are acceptable if you discard "Alakai the cunnining" or "cardinal Dominic" with "the shillellagh".
1 action and 5DP to deploy + 2 actions are too expensive for destroy "noted collector" or "hideout". :runner:

The cost for deploy a warrior : [x]DP + 1 action
The cost for deploy a support : 1 or 2 actions

With only one "curse of alegroth" you can easily discard 3 support cards in the same turn. 
With only one "Equilibrium" you can easily discard 2 support cards in the same turn. 
For 1 tactical action you can discard 1 support card with "DPAT-9".
These cards could discard "Bauhaus great infurior".

**Reply by Sol-au-fein** [reply]
Yeah, i know. Compared to other tools, it is super expensive. So lets just go with 2 actions deploy, 2 actions destroy. No DP cost. 
Or 2 actions deploy, 1 action + 3DP destroy.

**Comment by Sol-au-fein** [comment]
Lets finish Big Bertha:

1) 2 actions deploy, 2 actions destroy. No DP cost.

2) 2 actions deploy, 1 action + 3DP destroy.

**Reply by 21-ric** [reply]
For @TomekPL-1 this card is playable. So what do you think Tomek?

**Reply by TomekPL-1** [reply]
As I said before --> I use this card in current form but I don't mind if you decide to make it easier/cheaper to use.  Bauhaus is faction I like most since I started to collect doomtrooper cards. In my opinion the more improvements to this faction the better.

**Reply by Sol-au-fein** [reply]
I vote for 1) 2 actions deploy, 2 actions destroy. Seems like reusable Supply Lines Severed. 
Express yourself, so we can close discussion and update cards.

**Reply by 21-ric** [reply]
I understand @TomekPL-1,  you like this card like @Kite80 like "citadel of algeroth". 😊
Bauhaus is a good choice for a beginner.
In my opinion all useless cards must have an update.
"Supply lines severed" is better than "Bauhaus great infurior".
Same vote as @Sol-au-fein .

**Comment by TomekPL-1** [comment]
Bauhaus Great Infurior (BAUHAUS) [R]
INFLUENCE
Play for 1 Action and 6 [DP]. As 2 Actions, you may discard any SUPPORT. As 3 actions it may annihilate any SUPPORT.
